src/containers/HeaderContainer/store/actions.js

```diff
@@ -13,7 +13,7 @@ import { fetchTournamentUserData } from 'routes/Scoreboard/store'
  * @param {function} dispatch
  * @param {function} getState
  */
-const requestCollateralTokenBalance = account => (dispatch, getState) => {
+export const requestCollateralTokenBalance = account => (dispatch, getState) => {
```

Comment:
I don't think this is the best place for this. Can we maybe move this into something like `/store/actions/account/tokens`? wdyt?

Reply:
@andre-meyer I agree